modal/runner/engines/vllm.py

```diff
@@ -70,70 +82,56 @@ def __init__(self, params: VllmParams):
     async def generate(self, payload: CompletionPayload, params):
         assert self.engine is not None, "Engine not initialized"
 
+        # Track usage as a running total
+        # NOTE: This does NOT yet include cold-start GPU time
```

Comment:
how hard is this to add @sambarnes ?

Reply:
added in recent commits -- just tacks on the cold start time to the first request that started the container

my understanding is that this shouldn't have any thread safety issues since the `generate()` function is async so only one thread used